showcase/src/styles/_variables.scss

```diff
@@ -1,7 +1,11 @@
-/*$stark-typography-theme: (
-  font-family: '"Oswald", sans-serif',
+/*
+@import url('https://fonts.googleapis.com/css?family=Roboto');
```

Comment:
Can you expand this comment to also mention the other alternative about defining the `font-face`?

Can you add this comment to the Starter as well? So that everyone starting with Stark can see this ;)